Libraries Tasmania.js

```diff
@@ -0,0 +1,781 @@
+
+	// Dates have many different labels, loop through this list until one is found
+	// then save to item.date
+	// Dates will also be saved to Extra (see below) with their specific labels.
+	const dateLabels = [
+		"NI_YEAR_DISPLAY",
+		"NI_ADMISS_DATE",
+		"NI_TRIAL_DATE",
+		"NI_BIRTH_DATE",
+		"NI_DEATH_DATE",
+		"NI_MARRIAGE_DATE",
+		"NI_ARRIVAL_DATE",
+		"NI_DEPARTURE_DATE",
+		"NI_INQUEST_DATE",
+		"NI_LAND_DATE",
+		"NI_DOC_HEALTH_DATE",
+		"NI_EMPLOY_DATE",
+	];
+	for (let i = 0; i < dateLabels.length; ++i) {
+		let displayDate = getFieldText(doc, dateLabels[i]);
+		if (displayDate) {
+			item.date = displayDate;
+			break;
+		}
+	}
+
+	// Name index records have many possible fields.
+	// This list was made by scanning sample records, so labels could be missing.
+	// Loop through this list, and if values exist save them to Extra.
+	const fields = [
+		"NI_OCCUP",
+		"NI_YEAR_DISPLAY",
+		"NI_SHIP_NATIVE_PLACE",
+		"NI_REMARKS",
+		"NI_GENDER",
+		"NI_MOTHER",
+		"NI_P_OCCUP",
+		"NI_BIRTH_DATE",
+		"NI_BAPTISM_DATE",
+		"NI_REG_PLACE",
+		"NI_REG_YEAR",
+		"NI_AGE",
+		"NI_DEATH_DATE",
+		"NI_SPOUSE",
+		"NI_SPOUSE_GENDER",
+		"NI_SPOUSE_AGE",
+		"NI_MARRIAGE_DATE",
+		"NI_NAME_RANK",
+		"NI_DEPARTURE_DATE",
+		"NI_DEPARTURE_PORT",
+		"NI_SHIP",
+		"NI_BOUND",
+		"NI_WILL_NO",
+		"NI_PAGE",
+		"NI_NAME_TITLE",
+		"NI_ARRIVAL_DATE",
+		"NI_VOYAGE_NO",
+		"NI_CON_IDX",
+		"NI_SHIP_OR_FREE1",
+		"NI_NAME2",
+		"NI_SHIP_OR_FREE2",
+		"NI_MP_DATE",
+		"NI_CORONER",
+		"NI_DESC",
+		"NI_INQUEST_DATE",
+		"NI_VERDICT",
+		"NI_LAND_DATE",
+		"NI_LOCATION",
+		"NI_DOC_HEALTH_DATE",
+		"NI_UNDER14",
+		"NI_CENSUS_DISTRICT",
+		"NI_NAME_STATUS",
+		"NI_TRIAL_DATE",
+		"NI_OFFENSE",
+		"NI_VERDICT",
+		"NI_PP_ID",
+		"NI_EMPLOYER",
+		"NI_PROPERTY",
+		"NI_EMPLOY_DATE",
+		"NI_NOMINATOR",
+		"NI_DOC_DATE",
+		"NI_FATHER",
+		"NI_P_OCCUP",
+		"NI_PRE_SCHOOL",
+		"NI_ADMISS_DATE"
+	];
+	// Add field values to Extras
+	for (let i = 0; i < fields.length; ++i) {
+		let label = doc.querySelector("div." + fields[i] + "_label");
+		if (label) {
+			item[ZU.trim(label.textContent).replace(/:$/, "")] = getFieldText(doc, fields[i]);
+		}
+	}
+	item = addPermalink(doc, item, "NI");
+	addDigitalFiles(doc, item);
+}
+
+// Zotero.debug(item);
+	
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://librariestas.ent.sirsidynix.net.au/client/en_AU/tas/search/detailnonmodal/ent:$002f$002fARCHIVES_SERIES$002f0$002fARCHIVES_SER_DIX:AD940/one",
+		"items": [
+			{
+				"itemType": "manuscript",
+				"title": "Minutes of Meetings of Council",
+				"creators": [
+					{
+						"lastName": "University of Tasmania (TA92)",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "1890-01-19/1992-07-10",
+				"abstractNote": "Official minutes of meetings. | These records are part of the holdings of the Tasmanian Archives",
+				"callNumber": "AD940",
+				"libraryCatalog": "Libraries Tasmania",
+				"manuscriptType": "Series",
```

Comment:
I don't know if these really belong in `manuscriptType`... Maybe a line in `extra`?

Reply:
These are the type of archival entity being described -- series, item, or agency -- so I think `manuscriptType` is the best place for it. Having it there also makes it much easier to browse/filter different types in the Zotero interface. This is also consistent with other translators including National Archives of Australia and State Records Office of WA.

Reply:
I'm inclined to agree, but to make sure -- is this expected to be part of citations, typically? `manuscriptType` aka CSL's `genre` is used a fair amount in citation styles, so this would show up regularly. If that makes sense, I think this is definitely the right field. If it doesn't, I'd reconsider.

Reply:
I've made a couple of changes. I've kept `manuscriptType` to indicate the archival entity type ('series', 'item' etc). This is consistent with other translators (those noted above plus the UK National Archives) and is most likely to be used in citations. I've also lowercased these values for consistency.

I was also using `manuscriptType` for the type of index for names index records. I've changed these so the `manuscriptType` is now 'item', and the index title is saved in 'Extras'.